web/forms.py

```
@@ -564,6 +566,29 @@ def clean(self):
+        }
+        
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+            # Order subjects by name
+            self.fields['category'].queryset = Subject.objects.all().order_by('order', 'name')
+            
+        def clean_video_url(self):
+            url = self.cleaned_data.get('video_url')
+            if url:
+                # Simple validation for YouTube and Vimeo URLs
+                if not ('youtube.com' in url or 'youtu.be' in url or 'vimeo.com' in url):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)